localization/ndt_pcl_modified/include/ndt_pcl_modified/ndt.hpp

```diff
@@ -102,7 +102,11 @@ class NormalDistributionsTransformModified
   using NormalDistributionsTransform<PointSource, PointTarget>::step_size_;
   using NormalDistributionsTransform<PointSource, PointTarget>::gauss_d1_;
   using NormalDistributionsTransform<PointSource, PointTarget>::gauss_d2_;
+#if PCL_VERSION >= PCL_VERSION_CALC(1, 12, 1)
+  using NormalDistributionsTransform<PointSource, PointTarget>::point_jacobian_;
```

Comment:
Could you explain this a bit more in detail? :pray:

Reply:
You can see the api change in the first link in the description.

Reply:
I see, thank you!
![image](https://user-images.githubusercontent.com/31987104/165026332-c53e3b8f-b715-4080-ba93-7b26a839574e.png)